genesis/src/main.rs

```diff
@@ -462,7 +462,7 @@ fn main() -> Result<(), Box<dyn error::Error>> {
             ClusterType::Development => {
                 let hashes_per_tick =
                     compute_hashes_per_tick(poh_config.target_tick_duration, 1_000_000);
-                poh_config.hashes_per_tick = Some(hashes_per_tick);
+                poh_config.hashes_per_tick = Some(hashes_per_tick / 2);
```

Comment:
@sakridge what is this?

Reply:
That was experimenting with lowering the target rate to 1/2 the peak PoH rate to test in bench-tps. We can revert it for now.